### PR TITLE
Feature-73-project-settings-screen

### LINKED ISSUE #73: Project settings screen

### PR DESCRIPTION
**Pull request by lauriLukkarinen**
Resolves #73

### CODE REVIEW

File: src/routes/projects_.$projectId.settings.tsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+function ProjectSettingsScreen() {
+  const { t } = useTranslation();
+  const { projectId } = Route.useParams();
+  const { projectsApi } = useApi();
+  const queryClient = useQueryClient();
+
+  const findProjectQuery = useFindProjectQuery(projectId);
+  const project = useMemo(() => findProjectQuery.data, [findProjectQuery.data]);
+
+  const updateProject = useMutation({
```
Comment (Daniilmass) [inline src/routes/projects_.$projectId.settings.tsx:46]: Consider adding docs

File: src/routes/projects_.$projectId.settings.tsx [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    values: {
+      name: project?.name ?? "",
+      status: project?.status ?? ProjectStatus.Initiation,
+      estimatedStartDate: project?.estimatedStartDate ?? undefined,
+      estimatedEndDate: project?.estimatedEndDate ?? undefined,
+    },
+  });
+
+  console.log(getValidDateTimeOrThrow(project?.estimatedStartDate ?? new Date()));
+  console.log(getValidDateTimeOrThrow(project?.estimatedEndDate ?? new Date()));
```
Comment (Daniilmass) [inline src/routes/projects_.$projectId.settings.tsx:69]: Should these logs be removed?

File: src/routes/projects_.$projectId.settings.tsx [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+      } catch (error) {
+        throw new Error(t("errorHandling.errorUpdatingProject"), { cause: error });
+      }
+    },
+    onSuccess: () => {
+      queryClient.invalidateQueries({ queryKey: ["projects", projectId] });
+    },
+  });
+
+  const { control, handleSubmit, formState } = useForm<ProjectForm>({
```
Comment (Daniilmass) [inline src/routes/projects_.$projectId.settings.tsx:59]: Consider adding docs

**Review by Daniilmass — APPROVED** [review]

File: src/routes/projects_.$projectId.settings.tsx (hunk 2 above)
Comment (lauriLukkarinen) [reply src/routes/projects_.$projectId.settings.tsx]: Removed.

**Pull request merged**